src/start_server.ts

```diff
@@ -151,6 +156,32 @@ export function getApp(options: ServerOptions): express.Express {
 
   app.use('/components/', polyserve);
 
+  if (options.proxy) {
+    if (options.proxy.path === 'components') {
```

Comment:
`options.proxy.path.startsWith('components')`, and might need to do this after conditionally adding the leading `/`